### PR TITLE
Quickstart Flow Automation in Actions

### LINKED ISSUE #17: Quick start has daily pipeline run that goes through quick start flow
## Is your feature request related to a problem? Please describe.

At current, there is nothing validating that the docker-compose.yml in this file continues to work with the [quick start flow](https://www.conjur.org/get-started/)

## Describe the solution you would like

To ensure unexpected problems don't arise that break the quick start flow for end users, a pipeline should be added to this project (using github actions or jenkins) to run through the steps from the quick start and validate that they are functioning as expected. The pipeline should automatically run every day.

AC:
- [ ] This project has a daily build that runs through the quick start steps and alerts maintainers early of breaking changes

### PR DESCRIPTION
**Pull request by john-odonnell**
### What does this PR do?
Implementing a GitHub Action workflow to test the steps laid out in the Quick Start README and web tutorial.

### What ticket does this PR close?
Resolves #17 

#### Change log
- [ ] The CHANGELOG has been updated, or
- [x] This PR does not include user-facing changes and doesn't require a CHANGELOG update

#### Test coverage
- [x] This PR includes new unit and integration tests to go with the code changes, or
- [ ] The changes in this PR do not require tests

#### Documentation
- [ ] Docs (e.g. `README`s) were updated in this PR, and/or there is a follow-on issue to update docs, or
- [x] This PR does not require updating any documentation

### CODE REVIEW

File: .github/workflows/daily_test.yml [hunk 1]
```diff
@@ -0,0 +1,81 @@
+name: daily_flow_test
+on:
+  push
```
Comment (izgeri) [inline .github/workflows/daily_test.yml:3]: with this, will it run daily or just on updates to the repo / pushes?

Comment (john-odonnell) [reply .github/workflows/daily_test.yml:3]: The issue in question specifies it should be run daily, but for now it runs on pushes just so I have feedback on my test runs.

**Review by doodlesbykumbi — COMMENTED** [review]
This is a great addition.

I'm wondering if it might be better for us to lift all the execution steps into a shell script. My reasoning being that It would 
1. Make it possible to run the automation locally
2. Help with maintenance. Github Actions-driven dev can be a pain!

The resultant YAML would be a lot leaner.
```
name: daily_flow_test
on:
  schedule:
    # run once daily at 12 AM UTC, 8 PM EST
    - cron: '0 0 * * *'

jobs:
  # runs through all the tasks listed in conjur-quickstart daily
  # ensures that the quickstart flow is not broken for end users

  setup-conjur-env:
    runs-on: ubuntu-latest

    steps:
      - uses: actions/checkout@v2

      - name: Run the quick start flow
         run: ./bin/quick-start-flow
```

**Review by doodlesbykumbi — COMMENTED** [review]
Thanks for responding to my comments. I really like how `./test_workflow` is something anyone could pick up and run locally to get the same datapoint as CI.

I left a few comments on the new changes.

**Comment by diverdane** [comment]
This looks really good! Would you mind changing the name of `test_workflow` to `test_workflow.sh` (and make corresponding change where that file is called inside `.github/workflows/daily_test.yml`), and I'll approve.

**Review by diverdane — APPROVED** [review]
LGTM!!!

**Pull request merged**